app/web/features/messages/groupchats/GroupChatSendField.tsx

```diff
@@ -14,18 +15,26 @@ interface MessageFormData {
 
 export interface GroupChatSendFieldProps {
   sendMutation: UseMutationResult<string | undefined | Empty, RpcError, string>;
+  chatId: number;
+  currentUserId: number;
 }
 
 export default function GroupChatSendField({
   sendMutation,
+  chatId,
+  currentUserId,
 }: GroupChatSendFieldProps) {
   const classes = useSendFieldStyles();
 
   const { mutate: handleSend, isLoading } = sendMutation;
 
   const { register, handleSubmit, reset } = useForm<MessageFormData>();
+  const [persistedMessage, setPersistedMessage, clearPersistedMessage] =
+    usePersistedState(`messages.${currentUserId}.${chatId}`, "");
+
   const onSubmit = handleSubmit(async (data: MessageFormData) => {
     handleSend(data.text.trimRight());
+    clearPersistedMessage();
```

Comment:
I think it might be worth doing this when the user logs out as well, given our current privacy model. @lucaslcode thoughts?